src/scopen.js

```diff
@@ -15,13 +16,14 @@ const remotes = [
 
 export default options => {
   const { verbosity, cmd, application, file } = options;
-  const _branch = options.branch || getCurrentBranch();
+  const _cwd = dirname(file);
```

Comment:
I only used the underscores on `_branch` and `_remote` because I use `branch` and `remote` in the next closure scope down—I think this can just be `cwd`.